src/common/eeprom.cpp

```diff
@@ -431,6 +490,31 @@ static int eeprom_convert_from_v6(void) {
     return 1;
 }
 
+// conversion function for old version 8 (v 4.3.RC)
+static int eeprom_convert_from_v8(void) {
```

Comment:
Where do you copy current live-z offset into the first sheet when upgrading from previous FW?